Mathlib/Algebra/Star/Basic.lean

```diff
@@ -474,8 +476,10 @@ def starRingOfComm {R : Type*} [CommSemiring R] : StarRing R :=
 
 instance Nat.instStarRing : StarRing ℕ := starRingOfComm
 instance Rat.instStarRing : StarRing ℚ := starRingOfComm
+instance NNRat.instStarRing : StarRing ℚ≥0 := starRingOfComm
 instance Nat.instTrivialStar : TrivialStar ℕ := ⟨fun _ ↦ rfl⟩
 instance Rat.instTrivialStar : TrivialStar ℚ := ⟨fun _ ↦ rfl⟩
+instance NNRat.instTrivialStar : TrivialStar ℚ≥0 := ⟨fun _ ↦ rfl⟩
```

Comment:
The new instances could go in `Mathlib.Data.NNRat.Star` to avoid expanding imports.

Reply:
I think if the `Rat` instances are already here, the `NNRat` ones are fine to go next to them

Reply:
(But probably the right thing to do is move both)

Reply:
Yes, `Rat` and `NNRat` are very close in import weight (and will soon become even more since I want to define `NNRat.cast`), so no point avoiding `NNRat` but not `Rat`